src/shared/defaultAwsContextCommands.ts

```diff
@@ -410,4 +416,11 @@ export class DefaultAWSContextCommands {
         await this._awsContext.addExplorerRegion(profileRegion)
         this.refresh()
     }
+
+    private async getAccountId(): Promise<string | undefined> {
+        const client = ext.toolkitClientBuilder.createStsClient('us-east-1')
```

Comment:
I've got a couple locations where I'm prepopulating the region--I don't think STS needs a region, and `us-east-1` should work for all STS getClientIdentity calls. Not sure what the best thing to do is here.

Reply:
[JetBrains](https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ProjectAccountSettingsManager.kt#L125-L130) is using the default region `us-east-1` as well to access the global endpoint. But for a long term, we might need to consider using the regional endpoint as recommended [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html).